src/snippet/index.jsx

```diff
@@ -30,6 +30,10 @@ export const Snippet = ({ content, children, optional, fallback, ...props }) =>
   );
 };
 
-const mapStateToProps = ({ static: { content, establishment }, model }) => ({ content, establishment, model });
+const mapStateToProps = ({
+  static: { content, establishment, profile }, model
+}) => ({
+  content, establishment, profile, model
```

Comment:
On routes where the profile is not the model but the profile name is still in the breadcrumbs, (e.g. PILs) then we can't use the model prop and have to pass through the profile.

Reply:
also seems legit